src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+
+  if (days > 6) {
```

Comment:
also you can define constants for number of days after which the discount is applied

Reply:
agree